### PR TITLE
CLDR-13049 Fix TestAttributeValues plus data to make it pass.

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13049
- [ ] Updated PR title and link in previous line to include Issue number

Made some fixes to the test and DTDs so that the test passes, then turned the warning into an error. Also updated documentation for how to handle failures, and made the test point to that documentation.

See http://cldr.unicode.org/development/testattributevalues

Note that I found and fixed a bug when doing this: "segment" was not a valid bcp47 "nu" value.

### CODE REVIEW

**Comment by macchiati** [comment]
Ah, that exposed a different error. Will fix and resubmit.

Mark

**Comment by macchiati** [comment]
Tests pass now. Peter is out on Fridays (as am I in theory), can someone else approve?

File: common/main/en.xml [hunk 1]
```diff
@@ -1509,7 +1509,8 @@ annotations.
 			<type key="numbers" type="roman">Roman Numerals</type>
 			<type key="numbers" type="romanlow">Roman Lowercase Numerals</type>
 			<type key="numbers" type="saur">Saurashtra Digits</type>
-			<type key="numbers" type="shrd">Sharada Digits</type>
+            <type key="numbers" type="segment">Segmented Digits</type>
+            <type key="numbers" type="shrd">Sharada Digits</type>
 			<type key="numbers" type="sind">Khudawadi Digits</type>
```
Comment (pedberg-icu) [inline common/main/en.xml]: These use spaces instead of tabs but I won't this up for that reason...

**Comment by macchiati** [comment]
Thanks! The space v tab will be fixed by the cldr modify passes, but I agree that it is annoying (somehow I can't get the eclipse settings right).

**Pull request merged**